plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/LdapClient.java

```diff
@@ -17,7 +17,7 @@
 
 import java.util.Set;
 
-public interface LdapAuthenticatorClient
+public interface LdapClient
```

Comment:
What's the reason for renaming the interface?  Could add to PR description.

Reply:
I added an explanation in the PR description. Essentially, Authenticator is a type of Trino plugin, but we're re-purposing the client for Group Provider plugin as well.